platforms/android/library/src/main/java/io/element/android/wysiwyg/EditorStyledTextView.kt

```diff
@@ -13,23 +13,43 @@ import io.element.android.wysiwyg.view.StyleConfig
 import io.element.android.wysiwyg.view.inlinebg.SpanBackgroundHelper
 import io.element.android.wysiwyg.view.inlinebg.SpanBackgroundHelperFactory
 import io.element.android.wysiwyg.view.spans.ReuseSourceSpannableFactory
+import uniffi.wysiwyg_composer.MentionDetector
+import uniffi.wysiwyg_composer.newMentionDetector
 
 /**
  * This TextView can display all spans used by the editor.
  */
 open class EditorStyledTextView : AppCompatTextView {
 
+    private var mentionDetector: MentionDetector? = null
+
     private lateinit var inlineCodeBgHelper: SpanBackgroundHelper
     private lateinit var codeBlockBgHelper: SpanBackgroundHelper
-    private lateinit var styleConfig: StyleConfig
+
+    var styleConfig: StyleConfig? = null
```

Comment:
Same comment as above: [please can we either remove the nullability or handle it properly?